parmed/rdkit/rdkit.py

```diff
@@ -30,7 +30,9 @@ def load(rmol):
         from ..formats.pdb import PDBFile
         from rdkit import Chem
         fh = StringIO(Chem.MolToPDBBlock(rmol))
-        return PDBFile.parse(fh)
+        struct = PDBFile.parse(fh)
+        struct.coordinates = rmol.GetConformer().GetPositions()
```

Comment:
```suggestion
        struct.coordinates = rmol.GetConformer(0).GetPositions()
```

Reply:
I'm seeing this all now - I'm doing lots of ParmEd <--> rdkit conversions in order to use different functionalities in the same code. It looks like my old system of ad hoc copying bond orders no longer works with the new QualitativeBondType system -- I guess I'll need to figure this out.

The thing I would like to be able to do is read a PDB of a complex and then extract something like a ligand or cofactor as an rdkit object, but this causes trouble when I want rdkit to perceive bond orders itself.

Reply:
> The thing I would like to be able to do is read a PDB of a complex and then extract something like a ligand or cofactor as an rdkit object, but this causes trouble when I want rdkit to perceive bond orders itself.

Something like this should do it:

```python
import parmed as pmd
from rdkit import Chem

struct = pmd.load_file("your_file.pdb", expanded_residue_template_match=True)

ligand = struct[":LIG"]
ligand_rdkit = pmd.rdkit.RDKit.to_mol(ligand)
Chem.SanitizeMol(ligand_rdkit, catchErrors=True)
```

That should give you an RDKit-sanitized molecule pretty quickly and easily.  Note that `to_mol` does things the "right" way (that is, it instantiates an RWMol directly and adds the attributes via the API rather than going through PDB or something similar).